data-validation/use-cases.md

```diff
@@ -2,15 +2,148 @@
 
 ## Introduction
 
-Server-side data validation is an essential factor to the seamless interoperability promised by Solid. Two applications that manipulate the same kind of data, developed with no knowledge of each other, should be able to write that data without breaking each other. Using common linked data vocabularies is not enough to satisfy this fundamental need. Data Shape languages like [ShEx](http://shex.io/shex-semantics) and [SHACL](https://www.w3.org/TR/shacl/) help us solve this.
+Server-side data validation is an essential factor to the seamless interoperability promised by Solid. Multiple applications that manipulate the same kind of data, developed with no knowledge of each other, should be able to write that data without breaking each other. This requires all applications to conform to a common schema. Data Shape languages like [ShEx](http://shex.io/shex-semantics) and [SHACL](https://www.w3.org/TR/shacl/) help us solve this.
 
-Shapes allow us to define schemas with attributes from different linked data vocabularies. We can then test input data against those shapes and determine whether that input conforms to the shape or not. In other words, if we have a shape for a comment, and an application tries to write a comment, we can use our shape to validate whether that comment adheres to our expectation (the shape) of how a comment should be structured. Now we can use many different applications to safely read and write comments, just by adhering to the comment shape.
+Shapes allow us to define schema models with attributes from different linked data vocabularies. We can then test input data against those shape schemas and determine whether that input conforms to the shape or not. In other words, if we have a shape for a comment, and an application tries to write a comment, we can use that shape to validate whether that comment adheres to our expectation (the shape) of how a comment should be structured. Now we can use many different applications to safely read and write comments, just by adhering to the comment shape schema.
 
-Unfortunately, we cannot trust that all clients will write conformant data on their own. Consequently, we must be able to enforce conformance to a given shape on-pod, as the client writes the data.
+Schema validators are an enormous aid to application developers, but we cannot trust that all clients will write conformant data all the time. Consequently, we must be able to enforce conformance to a given shape at the pod, as the client writes the data.
+
+## Problem Cases
+
+The following use cases detail some of the problems related to a Solid ecosystem without on-pod data validation.
+
```

Comment:
The bullets in each problem case appear to be a sequence of events, the order of which is important. They should therefore be numbered (ordered) lists, instead of bulleted (unordered) lists.